code/android-core-library/src/test/kotlin/com/adobe/marketing/mobile/internal/utility/TimeUtilTest.java

```diff
@@ -19,6 +19,8 @@
 
 import static org.junit.Assert.*;
 
+import com.adobe.marketing.mobile.utils.TimeUtil;
```

Comment:
Must have missed moving the tests. Can we move this class to `test/java/com/adobe/marketing/mobile/utils` package?